code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -68,7 +68,14 @@ var/list/ai_list = list()
 			if (A.real_name == pickedName && possibleNames.len > 1) //fixing the theoretically possible infinite loop
 				possibleNames -= pickedName
 				pickedName = null
-
+	add_language("Robot Talk", 1)

```

Comment:
Actually don't know why i added this as its not a language. Its meant for binary actually.